sites/website/src/docs/fast-foundation/getting-started.md

```diff
@@ -6,22 +6,29 @@ custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website
 This document presumes you have [NodeJS](https://nodejs.org/) installed as well as some way to run an HTTP server locally, such as [http-server](https://www.npmjs.com/package/http-server).
 
 ## Quick-start
+
 This section will walk through the quickest way to get up and running using FAST components.
 
 ### Add the script
-Next, add the `fast-components.js` script file to your HTML. This pre-bundled script contains all components and dependencies. Also be sure to add [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to the script element.
+
+First, add the `fast-components.js` script file to your HTML. This pre-bundled script contains all components and dependencies. Also be sure to add [`type="module"`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) to the script element.
 
 ```html
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <script src="https://static.fast.design/assets/scripts/packages/fast-components.js" type="module"></script>
+        <script src="...TODO..." type="module"></script>
```

Comment:
I'm slightly confused by the "...TODO..." statement, though I think you're saying update this with your fast-components.js file wherever you have it hosted?  It may be better to use a relative path than a todo.  If this is pending update for the actual cdn location of /packages/version/fast-components.js when I get those versions published should we come back around and update this?

Reply:
Yes, this is a placeholder for the final CDN script location. I put "TODO" in a couple of places in the docs so I can find them all later and update it when we have the URL ready.